packages/driver/src/cross-origin/communicator.ts

```diff
@@ -113,6 +113,10 @@ export class PrimaryOriginCommunicator extends EventEmitter {
     debug('Unexpected postMessage:', data)
   }
 
+  isConnectedToSpecBridge (originPolicy: string) {
```

Comment:
I think we just want to call this origin @AtofStryker 

```suggestion
  isConnectedToSpecBridge (origin: string) {
```

Reply:
Realized I don't even use this method anymore so I removed it in https://github.com/cypress-io/cypress/pull/23643/commits/12277d6678d964cc1bf759ff834d7aeed13c0649